scripts/test-xgb-gpu-smoke.sh

```diff
@@ -21,7 +21,7 @@ function finish {
     set -e
 }
 
-./gradlew h2o-ext-xgboost:test &
+./gradlew h2o-ext-xgboost:test -x :h2o-ext-xgboost:testMultiNode &
```

Comment:
unnecessary colon
also the exclude of  h2o-ext-xgboost:multi seems unecessary as it depends on h2o-ext-xgboost:test and shoudl be trainsiently excluded no?

Reply:
`h2o-ext-xgboost:test` depends on `h2o-ext-xgboost:testSingleNode` and`h2o-ext-xgboost:testMultiNode` and others, i'm excluding multinode because i don't think we need to eun multinode in xgb smoke tests

Reply:
oh sorry, I misunderstood the line, makes sense